### PR TITLE
Add import project dialog UI component

### PR DESCRIPTION
**Pull request by pcardune**
This is a pure UI component for displaying an import project dialog. This is a piece of my applab import work. This component will get hooked up in a future diff.

import project dialog

### CODE REVIEW

File: apps/.storybook/head.html [hunk 1]
```diff
@@ -2,4 +2,5 @@
 <link rel="stylesheet" media="all" href="css/common.css">
 <link rel="stylesheet" media="all" href="css/applab.css">
 <link rel="stylesheet" media="all" href="css/droplet/droplet.min.css">
+<script src="https://use.fontawesome.com/8c9a5ecdc4.js"></script>
```
Comment (islemaster) [inline apps/.storybook/head.html:5]: Do we not have a local resource we can use (since this is not offline-friendly)?

Comment (pcardune) [inline apps/.storybook/head.html:5]: I looked for one and amazingly didn't find one. Not sure how it's getting loaded in the first place. Somewhere deep inside some ruby code I imagine.

File: apps/src/applab/ImportProjectDialog.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  propTypes: Object.assign({}, Dialog.propTypes, {
+    onImport: React.PropTypes.func.isRequired,
+    isFetching: React.PropTypes.bool,
+    error: React.PropTypes.bool,
+  }),
+
+  getDefaultProps() {
+    return {
+      isFetching: false,
+      error: false,
```
Comment (islemaster) [inline apps/src/applab/ImportProjectDialog.jsx]: Are these more or less redundant for non-required boolean props?

File: apps/src/applab/ImportProjectDialog.jsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+          <p style={styles.instructions}>
+            Copy the share link of the app you would like to import screens
+            from. Paste in the URL of that app below and click "Next."
+            {' '}<a href="#">Learn More</a>
+          </p>
+          <div style={styles.urlInputWrapper}>
+            <input type="text"
+                   value={this.state.url}
+                   style={styles.urlInput}
+                   onChange={event => this.setState({url: event.target.value})}/>
```
Comment (islemaster) [inline apps/src/applab/ImportProjectDialog.jsx]: Nit: Our styleguide says this should be

``` jsx
<input
    type="text"
    value={this.state.url}
    ...
/>
```

File: apps/src/applab/ImportProjectDialog.jsx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    storybook
+      .storiesOf('ImportProjectDialog', module)
+      .addStoryTable([
+        {
+          name: 'On open',
+          story: () => (
+            <ImportProjectDialog
+                hideBackdrop={true}
+                onImport={storybook.action("onImport")}
+                isFetching={false} />
```
Comment (islemaster) [inline apps/src/applab/ImportProjectDialog.jsx]: Don't know what we generally prefer; personally I like the shorthand for 'flag'-style boolean props:

``` jsx
<ImportProjectDialog
    hideBackdrop
    onImport={storybook.action("onImport")}
/>
```

**Comment by islemaster** [comment]
LGTM

File: apps/src/applab/ImportProjectDialog.jsx (hunk 2 above)
Comment (pcardune) [inline apps/src/applab/ImportProjectDialog.jsx]: well, I think in theory if they are not passed in, they will be null. I suppose the way they are being used makes that perfectly fine.

**Pull request merged**